app/views/spree/admin/overview/single_enterprise_dashboard.html.haml

```diff
@@ -47,14 +47,15 @@
     .header
       %h3
         %span.icon-map-marker
-        Your profile live
-      %p on the Open Food Network map
+          = t "your_profil_live"
+      %p
+        = t "on_ofn_map"
     .list
       /-# Can we pass an anchor here to zoom to our enterprise?
       %a.button.bottom{href: main_app.map_path, target: '_blank'}
-        See
+        = t "see"
```

Comment:
Can you give a link to the reference?

The [rails guide](http://guides.rubyonrails.org/i18n.html) has varying examples as well.

```ruby
t(:hello_flash)
```

There is an erb example which is very similar to haml. The second one is with parameters.
```erb
<h1><%=t :hello_world %></h1>
<%=t 'greet_username', user: "Bill", message: "Goodbye" %>
```

If you want to use namespaces within the key, you have to quote like this:
```ruby
t 'activerecord.errors.messages.record_invalid'
```

I would say: Parenthesis `()` never hurt, but you can omit them where they are not needed. Quoting the key `'name'` is more consistent, but I wouldn't bother to correct symbols like `:name`. The ideal would be `t('name')`.